tests/unit/Rules/UuidTest.php

```diff
@@ -25,27 +29,24 @@
 #[CoversClass(Uuid::class)]
 final class UuidTest extends RuleTestCase
 {
-    private const UUID_VERSION_1 = 'e4eaaaf2-d142-11e1-b3e4-080027620cdd';
-    private const UUID_VERSION_3 = '11a38b9a-b3da-360f-9353-a5a725514269';
-    private const UUID_VERSION_4 = '25769c6c-d34d-4bfe-ba98-e0ee856f3e7a';
-    private const UUID_VERSION_5 = 'c4a760a8-dbcf-5254-a0d9-6a4474bd1b62';
-
-    #[Test]
-    public function itShouldThrowExceptionWhenVersionIsTwo(): void
-    {
-        self::expectException(InvalidRuleConstructorException::class);
-        self::expectExceptionMessage('Only versions 1, 3, 4, and 5 are supported: 2 given');
-
-        new Uuid(2);
-    }
+    private const ALL_VERSIONS = [
```

Comment:
I changed how the tests were structure, so we don't need to use introspection to run tham.